src/modules/login/components/login_form.jsx

```diff
@@ -1,12 +1,16 @@
+import SystemSelector from './systemSelector';
+
 class LoginForm extends React.Component {
 
 	constructor(props) {
 		super(props);
 
 		this.state = {
 			email: '',
-			password: ''
-		}
+			password: '',
+			systemId: '0000d186816abba584714c92'	// the id of the 'local' system.
```

Comment:
We should discuss with @Langleu how it should work for the local system as it isn't related to any school